src/bidiMapper/domains/session/SessionProcessor.ts

```diff
@@ -26,15 +27,37 @@ import type {EventManager} from './EventManager.js';
 
 export class SessionProcessor {
   #eventManager: EventManager;
+  #browserCdpClient: CdpClient;
 
-  constructor(eventManager: EventManager) {
+  constructor(eventManager: EventManager, browserCdpClient: CdpClient,) {
     this.#eventManager = eventManager;
+    this.#browserCdpClient = browserCdpClient;
   }
 
   status(): Session.StatusResult {
     return {ready: false, message: 'already connected'};
   }
 
+  async create(params: Session.NewParameters): Promise<Session.NewResult> {
+    // Since mapper exists, there is a session already.
+    // Still the mapper can handle capabilities for us.
+    // Currently, only Puppeteer calls here but, eventually, every client
+    // should delegrate capability processing here.
+    const version = await this.#browserCdpClient.sendCommand('Browser.getVersion');
+    return {
+      sessionId: 'unknown',
+      capabilities: {
+        acceptInsecureCerts: false,
```

Comment:
Can we get this value from the options?

Reply:
Not really, since we would need to implement the capability matching algorithm. I will look into that later since it is not needed for Puppeteer.